frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1252,21 +1259,35 @@ export const surveyLogic = kea<surveyLogicType>([
                     const questionResults = surveyRatingResults[questionIdx]
 
                     // If we don't have any results, return 'No data available' instead of NaN.
-                    if (questionResults.total === 0) {
+                    if (questionResults.total === 0 || !questionResults) {
```

Comment:
@lucasheriques, wouldn't that break? the `!questionResults` has to be checked first and it should be `&&`

Reply:
follow up here https://github.com/PostHog/posthog/pull/29187